bom/pom.xml

```diff
@@ -59,7 +59,7 @@ THE SOFTWARE.
       <dependency>
         <groupId>org.apache.ant</groupId>
         <artifactId>ant</artifactId>
-        <version>1.9.14</version>
+        <version>1.10.7</version>
```

Comment:
This change breaks `mvn hpi:run` in plugins depending on Jenkins 2.230+ with JDK 8 with the error:

> `Error resolving project artifact: Failure to transfer com.sun:tools:pom:1.8.0` [...]

Reply:
We can probably exclude that dependency. It will take more research. Have you filed a Jira ticket or anything @daniel-beck ?

Reply:
actually ant doesn't declare this dependency at all. Could this come from the fact that this version of ant was compiled with java11 though with the class versions for java8? If yes we might go back to ant 1.10.5. The reason for using newer ant was that since version 1.10.2 (might also be 1.10.3) ant is able to properly handle symlinks on windows.

Reply:
1.10.5 looks like it works from my quick tests. That's a reasonable explanation for where the extra dependency comes from.

Reply:
Filed [JENKINS-61937](https://issues.jenkins-ci.org/browse/JENKINS-61937) just now for this.

Weirdly it looked like the dependency exists when I `dependency:tree` the plugin with the Jenkins dependency.